tgstation.dme

```diff
@@ -1487,6 +1487,7 @@
 #include "code\modules\vehicles\lawnmower.dm"
 #include "code\modules\vehicles\segway.dm"
 #include "code\WorkInProgress\atlantique\screambot.dm"
+#include "code\WorkInProgress\crystal\pixelprojectiles.dm"
```

Comment:
pretty sure its literally a work in progress seeing how vadim never perfected this and wanted to revisit it iirc